testapi.pm

```diff
@@ -108,6 +108,31 @@ os-autoinst is used in the openQA project.
 
 =head1 internal
 
+=head2 calculate_clickpoint
+
+This is used to support several subroutines dealing with mouse clicks, do not call
+this from the tests.
+
+=cut    
+
+sub calculate_clickpoint {
```

Comment:
Other internal functions within this module already use the convention to start the name with `_` so I suggest to stick with that convention.

Reply:
Ok, this is my very first PR, so I did not know, what exactly is requested. I will try to fix what I am able to.